internal/model/user_raw.go

```diff
@@ -16,6 +16,44 @@ type RawUserData struct {
 	CourseLMSID string `json:"course-lms-id" help:"LMS ID for the new user in the specified course."`
 }
 
+// Raw/dirty data for a course user.
+// This struct is used for raw data coming from a single course.
+type RawCourseUserData struct {
```

Comment:
Since we have "RawCourseUserData",
we should probably rename "RawUserData" to "RawServerUserData".